src/cpp/src/block_manager.hpp

```diff
@@ -6,20 +6,25 @@
 #include <memory>
 #include <list>
 #include <map>
+#include <chrono>
 
 #include "sequence_group.hpp"
 
 namespace ov::genai {
 class KVCacheBlock {
     int m_ref_count;
     int m_index;
+    size_t m_hash;
+    size_t m_num_hashed_tokens;
+    std::chrono::time_point<std::chrono::system_clock> m_timestamp;
 public:
     using Ptr = std::shared_ptr<KVCacheBlock>;
     using CPtr = std::shared_ptr<const KVCacheBlock>;
 
     explicit KVCacheBlock(int index)
         : m_ref_count(0),
-          m_index(index) { }
+          m_index(index),
```

Comment:
should `m_num_hashed_tokens` be set to zero here?

Reply:
Removed `m_num_hashed_tokens` in PR: https://github.com/openvinotoolkit/openvino.genai/pull/758/files